java/src/main/java/com/connexta/transformation/TransformApiDelete.java

```diff
@@ -0,0 +1,38 @@
+package com.connexta.transformation;
+
+import com.connexta.transformation.rest.models.ErrorResponse;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@Validated
+@Api(value = "transform", description = "the transform API")
```

Comment:
I'm not sure if the `Validated` annotation is necessary or if the `Api` annotation should be as well. Also not sure if the value and description should be different for each interface.

Reply:
Yes for `@Validated` and for the description, it ain't used right now so don't tweak it